docs/meshstack.user-group-provisioning.md

```diff
@@ -143,6 +143,33 @@ To set up the provisioning on AAD side, have a look at [Microsoft's guideline](h
 | Join(" ", [givenName], [surname])                             | name.formatted                             |
 | Attribute which is used as euid in your meshStack             | externalId                                 |
 
+### How to map attributes via expressions
```

Comment:
In many situations it can be useful to modify user attributes before syncing them to meshStack. For example...